modules/password.sh

```diff
@@ -2,16 +2,22 @@ function password {
   checkrpi
   checkroot
   checkargn $# 1
-  if [[ $1 == "" ]]; then
+
+  case "$1" in
```

Comment:
@JLKwong good style is to introduce a variable called `$options`

Reply:
I replaced all instances of `$1` with `$options`.